### DISCUSSION TITLE
Error: Minified React error #294 NextJS Static generation

### DISCUSSION THREAD

**Discussion by prrrcl** [pmndrs/jotai#588] [Q&A]
Hi all!

I have a Suspense on top of my component tree. Like an example that you give us.
````
const suspenseProps: any = isSSR ? {} : { fallback: <Loader isLoading /> };
````

but i got: Error: Minified React error #294; visit https://reactjs.org/docs/error-decoder.html?invariant=294 

after the compiled successfully message, with static pages generation

any idea?

**Comment by dai-shi** [comment]
I'm not sure if I get the situation, but `<Suspense>` is not yet supported in SSR, and you want to somehow avoid suspending in SSR. One way is to use Provider's `initialValues`, so that the atom won't suspend.

Wish someone can help on this?

**Reply by prrrcl** [reply]
The suspense component is that: `const SsrSuspense = isSSR ? Fragment : Suspense;`, I will try to give the initial values.

**Reply by prrrcl** [reply]
I had a problem with a specific file, a page.

The problem was generating the static. The solution I have given you is to force the ssr.

**Reply by dai-shi** [reply]
I see. I should have asked where you use async atoms. They require `<Suspense>`, so avoiding async atoms is the first thing to try.
